mqemitter-redis.js

```diff
@@ -18,8 +18,8 @@ function MQEmitterRedis (opts) {
 
   this._opts = opts
 
-  this.subConn = new Redis(opts)
-  this.pubConn = new Redis(opts)
+  this.subConn = new Redis(opts.connectionString || opts)
+  this.pubConn = new Redis(opts.connectionString || opts)
```

Comment:
Can you please add a unit test for this?

Reply:
sure thing

Reply:
@mcollina is this PR good to go?